matrix/commands.py

```diff
@@ -1321,6 +1342,64 @@ def predicate(event_id, line):
     return W.WEECHAT_RC_OK
 
 
+@utf8_decode
+def matrix_reply_command_cb(data, buffer, args):
+    def predicate(event_id, line):
+        event_tag = SCRIPT_NAME + "_id_{}".format(event_id)
+        tags = line.tags
+
+        if event_tag in tags:
+            return True
+        return False
+
+    for server in SERVERS.values():
+        if buffer in server.buffers.values():
+            room_buffer = server.find_room_from_ptr(buffer)
+
+            # event_id, reply = parse_reply_args(args)
+            event_id, reply = parse_redact_args(args)
```

Comment:
This is going to have to change, I've just taken advantage of `parse_redact_args` for now, perhaps erroneously.